exercises/baffling-birthdays/canonical-data.json

```
@@ -0,0 +1,164 @@
+      "description": "matching birthday for same date",
+      "property": "matchingBirthday",
+      "input": {
+        "birthdate1": "2000-01-01",
+        "birthdate2": "2000-01-01"
+      },
+      "expected": true
+    },
+    {
+      "uuid": "7193409a-6e16-4bcb-b4cc-9ffe55f79b25",
+      "description": "matching birthday for same year and month but different day",
+      "property": "matchingBirthday",
+      "input": {
+        "birthdate1": "2012-05-09",
+        "birthdate2": "2012-05-17"
+      },
+      "expected": false
+    },
+    {
+      "uuid": "d04db648-121b-4b72-93e8-d7d2dced4495",
+      "description": "matching birthday for same month and day but different year",
+      "property": "matchingBirthday",
+      "input": {
+        "birthdate1": "1999-10-23",
+        "birthdate2": "1988-10-23"
+      },
+      "expected": true
+    },
+    {
+      "uuid": "3c8bd0f0-14c6-4d4c-975a-4c636bfdc233",
+      "description": "matching birthday for same year but different month and day",
+      "property": "matchingBirthday",
+      "input": {
+        "birthdate1": "2007-12-19",
+        "birthdate2": "2007-04-27"
+      },
+      "expected": false
+    },
+    {
+      "uuid": "df5daba6-0879-4480-883c-e855c99cdaa3",
+      "description": "matching birthday for different year, month and day",
+      "property": "matchingBirthday",
+      "input": {
+        "birthdate1": "1997-08-04",
+        "birthdate2": "1963-11-23"
+      },
+      "expected": false
+    },
+    {
+      "uuid": "70b38cea-d234-4697-b146-7d130cd4ee12",
+      "description": "random birthdates generates requested number of birthdays",
+      "scenarios": ["random"],
+      "property": "randomBirthdates",
+      "input": {},
+      "expected": "length == groupsize"
+    },
+    {
+      "uuid": "d9d5b7d3-5fea-4752-b9c1-3fcd176d1b03",
+      "description": "random birthdates have non-leap year",
+      "scenarios": ["random"],
+      "property": "randomBirthdates",
+      "input": {},
+      "expected": {
+        "years": {
+          "leapYear": false
+        }
+      }
+    },
+    {
+      "uuid": "d1074327-f68c-4c8a-b0ff-e3730d0f0521",
+      "description": "random birthdates have random month",
+      "scenarios": ["random"],
+      "property": "randomBirthdates",
+      "input": {},
+      "expected": {
+        "months": {
+          "random": true
+        }
+      }
+    },
+    {
+      "uuid": "7df706b3-c3f5-471d-9563-23a4d0577940",
+      "description": "random birthdates have random day",
+      "scenarios": ["random"],
+      "property": "randomBirthdates",
+      "input": {},
+      "expected": {
+        "days": {
+          "random": true
+        }
+      }
+    },
+    {
+      "uuid": "ade37c87-f41d-4929-962a-286b4d1d048a",
+      "description": "has matching birthdays with matching birthday pair",
+      "property": "hasMatchingBirthdays",
+      "input": {
+        "birthdays": ["1970-01-19", "1975-06-03", "2003-01-19"]
```

This looks like it should have been birthdates
```suggestion
        "birthdates": ["1970-01-19", "1975-06-03", "2003-01-19"]
```